Firestore/core/src/core/view.h

```diff
@@ -182,6 +182,23 @@ class View {
       const core::ViewDocumentChanges& doc_changes,
       const absl::optional<remote::TargetChange>& target_change);
 
+  /**
```

Comment:
Optional: Instead of adding a new overload of `View::ApplyChanges()` you could just add the new parameter to the method signature and assign it a default value. Totally up to you.

Reply:
Sounds good. `ApplyChanges` already has 2 overloads. To keep it consistent, removed all overloads and give default value to both `const absl::optional<TargetChange>&` and `bool targetIsPendingReset`